tools/webpack/blocks.js

```diff
@@ -140,7 +143,11 @@ module.exports = {
 							// block will still call the core function when updates are back ported.
 							content = content.replace(
 								new RegExp( prefixFunctions.join( '|' ), 'g' ),
-								( match ) => `${ prefix }${ match }`
+								( match ) =>
+									`${ prefix }${ match.replace(
+										/^wp_/,
+										''
+									) }`
```

Comment:
I'm not super familiar with those files but isn't this already done in the same file below? 🤔  https://github.com/WordPress/gutenberg/blob/e21975baf35aae76d89d942a488668a70e696c5d/tools/webpack/blocks.js#L162-L169

Reply:
Oh, wait forget it! The snippet I linked is replacing inside of the function definitions. You were doing it in the function calls 🤦 .